### DISCUSSION TITLE
Review Club: Fuzz test

### DISCUSSION THREAD

**Discussion by VirdiJatin** [Bitshala/BitcoinCore-PR-Review-Club#115] [PR Reviews]

**Comment by VirdiJatin** [comment]
@stratospher Ping for video name and description...
@paperpsych Ping for thumbnail..

**Comment by paperpsych** [comment]
@VirdiJatin 
![Bitshala RC (110724)](https://github.com/user-attachments/assets/cccadc08-df5d-4f7a-a2fc-9cdb4a7c3a63)
